update.sh

```diff
@@ -31,6 +36,21 @@ if [ $? = 0 ]; then
 
 fi
 
-docker-compose stop
+cat -vt docker-compose.yml | egrep "  elastic:" > /dev/null
+
+if [ $? = 1 ]; then
+    echo "Your docker-compose don't contains Elastic section. Try to add."
+    echo "\\n\\n  elastic:\\n    image: elasticsearch:2" >> docker-compose.yml
+fi
+
+cat -vt docker-compose.yml | egrep "      - elastic" > /dev/null
+
+if [ $? = 1 ]; then
+    echo "Your docker-compose don't contains Elastic link to Web App container. Try to add."
```

Comment:
change to "Your docker-compose doesn't contain Elastic link in the Web App container. Trying to add."